CITATION.cff

```diff
@@ -28,9 +28,15 @@ authors:
 - given-names: "David"
   family-names: "Dotson"
   orcid: "https://orcid.org/0000-0001-5879-2942"
+- given-names: Joshua T.
+  family-names: Horton
+  orcid: 'https://orcid.org/0000-0001-8694-7200'
+- given-names: Matthew
+  family-names: Thompson
+  orcid: 'https://orcid.org/0000-0002-1460-3983'
 title: "The Open Free Energy library"
-version: 1.0
-date-released: 2024-04-19
+version: 1.1.0
+date-released: 2024-09-25
```

Comment:
Do I need to update the doi?

Reply:
pretty sure I have the DOI setup to point to "latest" I will check!

Reply:
From zenodo:

Cite all versions? You can cite all versions by using the DOI [10.5281/zenodo.8344247](https://zenodo.org/doi/10.5281/zenodo.8344247). This DOI represents all versions, and will always resolve to the latest one. [Read more](https://zenodo.org/help/versioning).

Reply:
so no need to update the DOI